pandas/tests/generic/test_series.py

```diff
@@ -227,3 +227,9 @@ def test_valid_deprecated(self):
         # GH18800
         with tm.assert_produces_warning(FutureWarning):
             pd.Series([]).valid()
+
+    def test_shift_always_copy(self):
+        # GH22397
+        s = Series([1, 2, 3])
+        assert s.shift(0) is not s
+        assert s.shift(1) is not s
```

Comment:
Can we parametrize this?  (I know it's only two examples, but why not three for example?)

Reply:
This seems fine to be as is.

Reply:
Done